sigmastate/src/main/scala/sigmastate/interpreter/Interpreter.scala

```diff
@@ -82,6 +115,7 @@ trait Interpreter extends ScorexLogging {
     case _ => None
   }
 
+  /** Helper class which encapsulates a mutable value. */
```

Comment:
Not good to put such utility class into middle of Interpreter. Should be moved to helper objects.

Reply:
moved to sigmastate.util.Helpers